README.md

```diff
@@ -58,9 +58,6 @@ See our [Migration Guide](https://google.github.io/accompanist/swiperefresh/) fo
 ### 🎨 [AppCompat Theme Adapter](./appcompat-theme/) (Deprecated)
 See our [Migration Guide](https://google.github.io/accompanist/appcompat-theme/) for migrating to the new artifact in Accompanist.
 
-### 📖 [Pager](./pager/) (Deprecated)
```

Comment:
For insets we did `(Deprecated & Removed)`, do we want to do the same for these other ones for a bit? Or just remove completely?